unit_tests/test_tester.py

```diff
@@ -196,6 +199,31 @@ def _validate_results(self):
         assert self.final_event.test_status == 'FAILED'
 
 
+class CriticalErrorNotCaughtTest(ClusterTesterForTests):
+    @staticmethod
+    def test():
```

Comment:
Is it really needed to be "staticmethod"? Not instance method as in all other tests here?

Reply:
Yes, otherwise pylint is swearing no-self-use, it is indeed weird that it does not do that on other  occasions.
But it is what I met with this particular case.